share/spack/qa/run-unit-tests

```diff
@@ -38,6 +38,7 @@ bin/spack help -a
 
 # Profile and print top 20 lines for a simple call to spack spec
 spack -p --lines 20 spec mpileaks%gcc ^dyninst@10.0.0 ^elfutils@0.170
+$coverage_run $(which spack) bootstrap status
```

Comment:
@scheibelp @tldahlgren I didn't add real unit tests since, given this is not a hot code path and it likely needs a lot of mockery to try different cases, I thought they would have been "low-revenue". Anyhow, I'll try to add them given your two requests.